content/using-bookwyrm/posting-statuses.md

```diff
@@ -0,0 +1,25 @@
+Title: Posting statuses
+Date: 2022-04-30
+Order: 1
+
+Posting on BookWyrm is similar to [posting on Mastodon](https://docs.joinmastodon.org/user/posting/) in that statuses have visibility settings, content warnings, and mentions. However, polls and attachments are not supported.
+
+## Types of posts
```

Comment:
I would refine this slightly - there are three ways to post _about a book_. It's probably not useful info for this page, but technically there are also replies and generated posts, so while I think this is the right framing and scope, I think just specifying that these are how to post about a book narrows it down?

Reply:
that makes sense! I updated it so it doesn't leave that out anymore